docs/components/statistic.njk

```diff
@@ -220,7 +220,7 @@ tags:
 {%- endcall %}
 
 <h4>Dark theme - secondary background</h4>
-<div class="example" style="background: #3C3F42; border-radius: 3px;">
+<div class="example" style="background: #212427; border-radius: 3px;">
```

Comment:
We have access to all the tokens, so why not use the semantic token name?

Reply:
Ha, good point. I ended up adding the tokens with their fallback values in the Accessibility section. @coreyvickery, if I pulled some inline styles from another doc page, I left it alone until the redesign. An example would be the helpful tip formatting that I found in the new nav pages.

Reply:
@marionnegp Approved.